### PR TITLE
Fix part of #34 & #35: Hifi State Button

### PR DESCRIPTION
**Pull request by rt4914**
## Explanation

This PR has one issue, we do not have a mechanism to make "Submit" button inactive. Will need help from @BenHenning regarding this.

https://xd.adobe.com/view/54bd2eee-0536-4504-4976-8e2211bd169a-228c/screen/40d6cf5f-f673-42c8-af15-035f711ef6b9/PM-IN-Introduction-

## Checklist

- [x] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
- [x] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
- [x] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
- [x] The PR does not contain any unnecessary auto-generated code from Android Studio.
- [x] The PR is made from a branch that's **not** called "develop".
- [x] The PR is made from a branch that is up-to-date with "develop".
- [x] The PR's branch is based on "develop" and not on any other branch.
- [x] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.

## Screenshots
Screenshot 2019-11-13 at 3 54 40 PM
Screenshot 2019-11-13 at 3 54 59 PM

### CODE REVIEW

File: app/src/main/res/layout/state_button_item.xml [hunk 1]
```diff
@@ -14,44 +14,52 @@
...
+    <ImageButton
       android:id="@+id/previous_state_image_view"
       android:layout_width="wrap_content"
       android:layout_height="wrap_content"
       android:layout_alignParentStart="true"
+      android:layout_margin="4dp"
       android:background="@drawable/previous_next_state_image_view_background"
       android:contentDescription="@string/previous_state_description"
-      android:padding="8dp"
+      android:elevation="2dp"
```
Comment (veena14cs) [inline app/src/main/res/layout/state_button_item.xml:31]: This elevation works above api 21. Please check.

**Comment by veena14cs** [comment]
Please add screenshot for reference.

File: app/src/main/res/layout/state_button_item.xml (hunk 1 above)
Comment (rt4914) [reply app/src/main/res/layout/state_button_item.xml:31]: I have mentioned that in High-Fidelity Mock Known issues. And also, this same case is appearing in nested recyclerview too, and therefore needs discussion with Ben.

**Comment by rt4914** [comment]
> Please add screenshot for reference.

Added screenshots.

**Review by veena14cs — APPROVED** [review]

**Pull request merged**